src/__experimental__/components/checkbox/checkbox.d.ts

```diff
@@ -29,6 +29,10 @@ interface CheckboxProps {
   Pass string to display icon, tooltip and blue border
   Pass true boolean to only display blue border */
   info?: boolean | string;
+  /** Margin bottom, given number will be multiplied by base spacing unit (8) */
+  mb?: 0 | 1 | 2 | 3 | 4 | 5 | 7;
+  /** Margin left as a percentage */
```

Comment:
The comment in `checkbox.component.js` states `any valid CSS value` but here it is a percentage?